sklearn/multiclass.py

```diff
@@ -915,7 +1068,7 @@ class OutputCodeClassifier(MetaEstimatorMixin, ClassifierMixin, BaseEstimator):
     classes_ : ndarray of shape (n_classes,)
         Array containing labels.
 
-    code_book_ : ndarray of shape (n_classes, code_size)
+    code_book_ : ndarray of shape (n_classes, len(estimators_))
```

Comment:
this is not true, the shape is correct as it was. And code_size is not the same as `len(estimators_)`